pkg/subctl/cmd/validate_tunnel.go

```diff
@@ -191,7 +191,15 @@ func getGatewayIP(remoteCfg *rest.Config, localClusterID string) string {
 
 		for _, conn := range gw.Status.Connections {
 			if conn.Endpoint.ClusterID == localClusterID {
-				return conn.UsingIP
+				if conn.UsingIP != "" {
```

Comment:
https://github.com/submariner-io/submariner-operator/pull/1507#pullrequestreview-711872457

Isn't this part a leak of the above PR ?

Reply:
Oh yes. As you might have noticed the directory structure and file-names are changed between release-0.9 and devel. So I could not do a clean cherry-pick and had to do manual changes. 

Just to be sure that changes are not causing regression, I manually built subctl and verified it locally on a KIND cluster. In that process, I was getting this error, so included it and test-cases passed. I forgot to remove it ;-) 

I'll update this PR and will remove this change. 

Thanks for reviewing @mangelajo